From 633ed4d425167afe0bf692d47c7767085247d871 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Sat, 18 Oct 2025 10:39:13 +0900
Subject: [PATCH] ci: fix workaround about bpftool for codeql

Follow-up for e9fd2bbfffc5c2c7cd1ea0a288d5435fc15e387f.

(cherry picked from commit a6836cfa0bdf1bb1fcf05686c5af3f2b5ad97f6b)

Related: RHEL-155394
---
 .github/workflows/codeql.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.github/workflows/codeql.yml b/.github/workflows/codeql.yml
index 443221fa11..cfe54d59ac 100644
--- a/.github/workflows/codeql.yml
+++ b/.github/workflows/codeql.yml
@@ -49,7 +49,7 @@ jobs:
         # TODO: drop after we switch to ubuntu 26.04
         bpftool_binary=$(find /usr/lib/linux-tools/ /usr/lib/linux-tools-* -name 'bpftool' -perm /u=x 2>/dev/null | sort -r | head -n1)
         if [ -n "$bpftool_binary" ]; then
-          sudo rm -f /usr/bin/bpftool
+          sudo rm -f /usr/{bin,sbin}/bpftool
           sudo ln -s "$bpftool_binary" /usr/bin/
         fi
 
